src/ca65/instr.c

```diff
@@ -1136,6 +1287,14 @@ static void PutPCRel16 (const InsDesc* Ins)
 
 
 
+static void PutPCRel4510 (const InsDesc* Ins)
+/* Handle branches with a 16 bit distance */
+{
+    /* 16 bit branch opcode is 8 bit branch opcode or 0x03 */
```

Comment:
Hmm, I think that the comment is a little confusing.  It looks as though you are saying that it's a choice between an original opcode and "0x03" (instead of the original one).  So, I think that it should be changed to
"... opcode or'ed with 0x03".